Soma-Technologies-Inc/visitAfrica-backend
# #175494073 Integrate the requests table

#### What does this PR do?
* this PR help users to get the trip request she/he have requested.
#### Description of Task to be completed?
* user or a requester will be able to get the whole information she have commanded with the status of his/her trip. by using the route of `http://localhost:3000/api/requests` in post man with the authorization to get her/his own request
* A user should see a list of requests that she/he have submitted over time to Barefoot nomad and their current status.
#### How should this be manually tested?
*  clone the RP by using `clone https://github.com/Soma-Technologies-Inc/visitAfrica-backend.git`
*  make sure you know where you cloned your RP and enter in it and run `code .`
*  run `npm install ` to install some packages you do not have
*  run `npm run startDev`
*  Loging and copy the token of yours to use in the following route
*  got to post man on the router of `http://localhost:3000/api/requests`
*  get your request
#### Any background context you want to provide?
N/A
#### Screenshots (if appropriate)
![reqpc1](https://user-images.githubusercontent.com/73880097/103929950-ae4fbb00-5126-11eb-88cc-f640b4ab4cec.PNG)

![reqpc2](https://user-images.githubusercontent.com/73880097/103929980-b90a5000-5126-11eb-9f3a-eca35dd203b8.PNG)

![reqpc3](https://user-images.githubusercontent.com/73880097/103930002-bf98c780-5126-11eb-8ef4-7a4acc86b016.PNG)

#### Questions:
